### PR TITLE
Add float content

### PR DESCRIPTION
**Pull request by kutsaniuk**
Component created for: 

image

https://www.figma.com/file/rA8iJo2alEEoOVTUxDwutA/Payment-Link?node-id=706%3A25385&mode=dev

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `OcFloatContent` Vue component for displaying floating content with dropdown options and close functionality. This component supports dynamic layout adjustments and customization of dropdown items.

- **Refactor**
	- Updated the package version to 0.5.86.
	- Added export for the `OcFloatContent` component for easier access and integration.

### CODE REVIEW

File: packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+const isOpen = ref(false);
+</script>
+
+<template>
+  <div
+    class="fixed top-0 bottom-0 shadow-xl bg-oc-bg-light transition-all h-full"
+    :class="{
+      'right-0': modelValue && position === 'right',
+      'right-[-999px] w-0': !modelValue && position === 'right',
```
Comment (HoaiLee) [inline packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue:48]: Can it be? apply for `position === 'left'` if possible

```suggestion
      'invisible': !modelValue && position === 'right',
```

File: packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      'left-0': modelValue && position === 'left',
+      'left-[-999px] w-0': !modelValue && position === 'left',
+    }"
+  >
+    <template v-if="modelValue">
+      <div
```
Comment (HoaiLee) [inline packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue:54]:
```suggestion
      <div
        v-if="modelValue"
```

File: packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+              v-if="isClose"
+              class="p-[6px] rounded cursor-pointer text-oc-gray-500 hover:bg-gray-100 hover:text-oc-text"
+              @click="$emit('update:model-value', false)"
+            >
+              <Icon name="x" width="20" height="20" />
+            </div>
+          </div>
+        </slot>
+      </div>
+      <slot name="default"></slot>
```
Comment (HoaiLee) [inline packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue:129]:
```suggestion
      <slot name="bottom" />
```

File: packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      'right-0': modelValue && position === 'right',
+      'right-[-999px] w-0': !modelValue && position === 'right',
+      'left-0': modelValue && position === 'left',
+      'left-[-999px] w-0': !modelValue && position === 'left',
+    }"
+  >
+    <div v-if="modelValue">
+      <div
+        class="flex border-oc-gray-200 gap-x-9 justify-between p-5 items-start border-b sticky bg-oc-bg-light top-0"
+      >
```
Comment (bdebashi) [inline packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue:56]: Is there a need for an extra `div` or can be as below + removing matching closing `div`
```suggestion
    <div v-if="modelValue" class="flex border-oc-gray-200 gap-x-9 justify-between p-5 items-start border-b sticky bg-oc-bg-light top-0">
```

**Review by bdebashi — APPROVED** [review]
Small comment for removing unnecessary `div`, otherwise LGTM

File: packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue (hunk 4 above)
Comment (kutsaniuk) [reply packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue:56]: @bdebashi We need because we hide 2 blocks -> header and slot

**Pull request merged**